src/lib/ui/core/Input/UniversalInput.svelte

```diff
@@ -0,0 +1,75 @@
+<script lang="ts">
+  import type { HTMLInputAttributes, HTMLTextareaAttributes } from 'svelte/elements'
+  import type { Snippet } from 'svelte'
+
+  import Svg from '$ui/core/Svg/index.js'
+  import { cn } from '$ui/utils/index.js'
+
+  type TCommonProps = {
+    class?: string
+    type?: 'text' | 'number' | 'email'
+    placeholder?: string
+    icon?: string
+    iconSize?: number | string
+    inputClass?: string
+    left?: Snippet
+    right?: Snippet
+  }
+
+  type TInputProps = {
+    as: 'input'
+  } & HTMLInputAttributes &
+    TCommonProps
+
+  type TTextAreaProps = {
+    as: 'textarea'
+  } & HTMLTextareaAttributes &
+    TCommonProps
+
+  type TProps = TInputProps | TTextAreaProps
+
+  let {
+    class: className,
+    inputClass,
+    type = 'text',
+    as,
+    placeholder = '',
+    icon,
+    iconSize = 12,
+    disabled,
+    left,
+    right,
+    ...rest
+  }: TProps = $props()
+
+  let inputEl = $state<HTMLInputElement | HTMLTextAreaElement>()
+</script>
+
+<!-- svelte-ignore a11y_click_events_have_key_events -->
+<!-- svelte-ignore a11y_no_static_element_interactions -->
+<div
+  class={cn(
+    'relative flex items-center rounded border bg-white fill-casper pl-2.5 pr-3 focus-within:border-green focus-within:fill-green hover:border-green',
+    disabled && 'bg-athens text-mystic hover:border-porcelain',
+    className,
+  )}
+  onclick={() => inputEl?.focus()}
+>
+  {#if icon}
+    <Svg id={icon} w={iconSize} class="mr-2.5" />
+  {/if}
+
+  {@render left?.()}
+
+  <svelte:element
+    this={as ?? 'input'}
+    bind:this={inputEl}
+    {...rest}
+    {disabled}
+    {type}
+    {placeholder}
+    class={cn('w-full rounded bg-transparent py-[5px] outline-none', inputClass)}
```

Comment:
Changed css structure (padding moved from `input` element to a parent container` breaks styles of other components that were applying custom styles via `inputClass` prop, e.g. `Gdpr.svelte`, `DigestForm.svelte`, `LabelInput.svelte` and some components on Sanbase.
This change will require additional updates of other components which are out of scope of alerts update. Better to return to the previous css styles structure to not cause cascade changes and apply this update in a different PR as a part of a new task